examples/with-npm/turbo.json

```diff
@@ -4,13 +4,14 @@
   "pipeline": {
     "build": {
       "dependsOn": ["^build"],
-      "outputs": ["dist/**", ".next/**", "!.next/cache/**"]
+      "outputs": [".next/**", "!.next/cache/**"]
     },
     "lint": {
-      "outputs": []
+      "dependsOn": ["^lint"]
```

Comment:
`^lint` is probably unnecessary. all lints can happen in parallel, there are no dependencies

Reply:
Ah, yeah, there's a difficulty here; I don't love it.

- `[]` means you can change `eslint-config` and still hit cache - problematic
- `["^lint"]` means not the best parellalization but at least it's correct - still not great
- Could use the `["^topo"]` technique here but feels a little too advanced for a starter.

What you think?

Reply:
Hmm, I see. I would use the `topo` config, since that's actually our recommended way of accomplishing this, even though it's confusing. IMO, we should show parallelism for tasks that don't depend on each other.